src/main/java/frc/robot/subsystems/ClimbingSubsystem.java

```diff
@@ -43,10 +48,19 @@ public ClimbingSubsystem(){
   public void climbControl(double leftSpeed, double rightSpeed){
     leftClimber.set(leftSpeed);
     rightClimber.set(rightSpeed);
+    if (leftSpeed != 0 || rightSpeed != 0) {
+      isClimbing = true;
+    }
   }
 
   @Override
   public void periodic(){
+    if (isClimbing = true) {
```

Comment:
This should be just: _if (isClimbing) {_

'=' is the assignment operator, and there's no need to use '==' when you're just checking a boolean.